htmresearch/algorithms/sensor_placement.py

```diff
@@ -0,0 +1,78 @@
+# ----------------------------------------------------------------------
```

Comment:
I don't think this belongs in algorithms. It is pretty specific - should probably go into frameworks/layers